dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css

```diff
@@ -5043,13 +5043,15 @@
   margin-top: 0.5rem;
 }
 .dijitComboBoxMenuPopup .dijitMenuItem {
+  padding: 0 0.75rem;
```

Comment:
Is there any `$space-x` variable that matches this value?

Reply:
same